interfaces/Portalicious/src/app/pages/project/project-team/add-user-button/add-user-button.component.html

```diff
@@ -1,7 +1,94 @@
 <p-button
-  label="Add user to team"
-  i18n-label
-  rounded
+  label="Add team member"
+  i18n-label="@@add-team-member"
+  outlined
```

Comment:
According to design:
```suggestion
  rounded
```

Reply:
> Lets see if we need to re-align to prevent future rework...

Yes, I'm curious what was the reasoning here too, but we can still go ahead and fix in the meantime.